src/Equinor.ProCoSys.IPO.WebApi/appsettings.json

```diff
@@ -12,5 +12,6 @@
   },
   "AllowedHosts": "*",
   "UseAzureAppConfiguration": false,
-  "EnableServiceBus": false
+  "EnableServiceBus": true,
+  "ServiceBus:TopicNames": "ipo" 
```

Comment:
Rename ServiceBus:TopicNames to ServiceBus:TopicNameList (or something) since you allow commas?